presto-main/src/main/java/com/facebook/presto/sql/analyzer/Analysis.java

```diff
@@ -435,31 +446,31 @@ public double getSampleRatio(SampledRelation relation)
         return sampleRatios.get(relation);
     }
 
-    public static class JoinInPredicates
+    public static class JoinSideExpressions
```

Comment:
It was "join side" expressions. But I have just removed this entity, see my previous comment. These expression were already known to `RelationPlanner` so there is no need to store it in `Analysis`.